src/sentry/monitors/logic/mark_failed.py

```diff
@@ -343,7 +343,9 @@ def get_monitor_environment_context(monitor_environment: MonitorEnvironment):
 def get_occurrence_data(checkin: MonitorCheckIn):
     if checkin.status == CheckInStatus.MISSED:
         expected_time = (
-            checkin.expected_time.strftime(SUBTITLE_DATETIME_FORMAT)
+            checkin.expected_time.astimezone(checkin.monitor.timezone).strftime(
```

Comment:
and if `timezone` is `None` this just shows `UTC`. cool.

Reply:
There should always be a timezone though, so we're good here.